README.md

```diff
@@ -1,21 +1,100 @@
-# flutter_bluetooth_serial
 
+# `flutter_bluetooth_serial`
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/e715d21e77394cfaacf9abd20b7d97cc)](https://app.codacy.com/app/edufolly/flutter_bluetooth_serial?utm_source=github.com&utm_medium=referral&utm_content=edufolly/flutter_bluetooth_serial&utm_campaign=Badge_Grade_Dashboard)
```

Comment:
Keep this.

Reply:
Well, if we want to keep this it must be updated for newest commit. Current score is B as far as I know (a lots of warns about markdown syntax lol). The other problem is also fact, that Dart isn't detected by the Codacy - so the whole badge is... irrelevant (because the library is written mostly in Dart).